### PR TITLE
Rails5

### PR DESCRIPTION
**Pull request by daegun**

### CODE REVIEW

File: lib/restful/jsonapi/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Restful
   module Jsonapi
-    VERSION = "0.0.13"
+    VERSION = "1.0.0"
```
Comment (zkwentz) [inline lib/restful/jsonapi/version.rb]: Can we go semantic versioning, eg. `0.1.0`?

File: restful-jsonapi.gemspec [hunk 2]
```diff
@@ -18,6 +18,8 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
+  spec.add_dependency "rails", "~>5.0.0.rc1"
```
Comment (zethussuen) [inline restful-jsonapi.gemspec]: Rails 5 is official as of 6/30/2016, so probably shouldn't use rc1?

File: lib/restful/jsonapi/version.rb (hunk 1 above)
Comment (daegun) [inline lib/restful/jsonapi/version.rb]: aram wanted a major version bump

File: restful-jsonapi.gemspec (hunk 2 above)
Comment (daegun) [inline restful-jsonapi.gemspec]: was just using rc1 for an app but yes we can bump

**Comment by mrryanjohnston** [comment]
👍

File: lib/restful/jsonapi/version.rb (hunk 1 above)
Comment (mrryanjohnston) [inline lib/restful/jsonapi/version.rb]: This should be our first major release. According to semver website:

http://semver.org/

>How do I know when to release 1.0.0?
>
>If your software is being used in production, it should probably already be 1.0.0. If you have a stable API on which users have come to depend, you should be 1.0.0. If you’re worrying a lot about backwards compatibility, you should probably already be 1.0.0.

**Pull request merged**